NU394-S2024TTh/savor
# In-app notification improvements

### Description

Changed original notification to modal

### QA

Please explain in detailed steps how the reviewer can test your changes and what the expected behaivors are (visual would help!). Consider different test cases/scenarios.

Click on the info icon of each row to display modal and then try closing it

### Packages

Please describe any additional packages you added to package.json.

### Linear Issue Tracking

Fixes SMA-50
Fixes SMA-27

### Checklist
